firebolt_provider/example_dags/example_firebolt.py

```diff
@@ -53,17 +53,13 @@
     default_args={"firebolt_conn_id": FIREBOLT_CONN_ID},
     tags=["example"],
     catchup=False,
+    template_searchpath=[PATH_TO_TEMPLATES],
 ) as dag:
-    firebolt_start_engine = FireboltStartEngineOperator(
-        task_id="firebolt_start_engine", engine_name=FIREBOLT_ENGINE
-    )
-    firebolt_stop_engine = FireboltStopEngineOperator(
-        task_id="firebolt_stop_engine", engine_name=FIREBOLT_ENGINE
-    )
-
+    firebolt_start_engine = FireboltStartEngineOperator(task_id="firebolt_start_engine")
+    firebolt_stop_engine = FireboltStopEngineOperator(task_id="firebolt_stop_engine")
     firebolt_op_sql_create_table = FireboltOperator(
         task_id="firebolt_op_sql_create_table",
-        sql=SQL_CREATE_TABLE_STATEMENT,
+        sql="firebolt_op_sql_create_table.sql",
```

Comment:
Is this change to indicate the users can now use sql files instead of strings? Let's add a comment above explaining why this case is different from others so it doesn't lead to confusion.